src/Template/Pages/Model/Categories/index.twig

```diff
@@ -10,10 +10,12 @@
 {% element 'Modules/index_header' %}
 
 <div class="module-index">
+    {% if schema.properties.categories %}
```

Comment:
With this `{⅜ if .. %}` I don't see any category... is it necessary?

Reply:
restored as before, this part